### PR TITLE
feat(argo-cd): Update to the latest redis-ha chart which defaults to redis 6.2.5

### PR DESCRIPTION
**Pull request by smcavallo**
See - https://github.com/DandyDeveloper/charts/tree/master/charts/redis-ha

Checklist:

* [X] I have bumped the chart version according to [versioning](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#versioning)
* [X] I have updated the chart changelog with all the changes that come with this pull request according to [changelog](https://github.com/argoproj/argo-helm/blob/master/CONTRIBUTING.md#changelog).
* [X] Any new values are backwards compatible and/or have sensible default.
* [X] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo).
* [X] My build is green ([troubleshooting builds](https://argoproj.github.io/argo-cd/developer-guide/ci/)).

Changes are automatically published when merged to `master`. They are not published on branches.

### CODE REVIEW

File: charts/argo-cd/Chart.yaml [hunk 1]
```diff
@@ -16,9 +16,9 @@ maintainers:
...
 dependencies:
   - name: redis-ha
-    version: 4.12.17
+    version: 4.13.0
     repository: https://dandydeveloper.github.io/charts/
     condition: redis-ha.enabled
 annotations:
   artifacthub.io/changes: |
-    - "[Changed]: Make AWS ALB GRPC backend protocol version configurable"
+    - "[Changed]: Update to the latest redis-ha chart which defaults to redis 6.2.5"
```
Comment (MichielDeMey) [inline charts/argo-cd/Chart.yaml:24]: I believe the changelog should append, not replace. 🙂

Comment (mbevc1) [reply charts/argo-cd/Chart.yaml:24]: It's only keeping changes relevant to this version, I believe this is okay

Comment (MichielDeMey) [reply charts/argo-cd/Chart.yaml:24]: Ah, I thought it meant a changelog per "major" verison, not "minor".

**Comment by mbevc1** [comment]
Seems upstream manifests are still using older image? https://github.com/argoproj/argo-cd/blob/master/manifests/base/redis/argocd-redis-deployment.yaml#L24

**Pull request closed without merging**